docs/src/pages/demos/snackbars/FabIntegrationSnackbar.js

```diff
@@ -96,31 +93,29 @@ class FloatingActionButtonZoom extends React.Component {
             <AddIcon />
           </Button>
           <Snackbar
-            open={this.state.open}
+            open={open}
             autoHideDuration={4000}
             onClose={this.handleClose}
             SnackbarContentProps={{
-              'aria-describedby': 'message-id',
-              className: classes.snackBarContent,
+              'aria-describedby': 'snackbar-fab-message-id',
+              className: classes.snackbarContent,
             }}
-            message={<span id="message-id">Archived</span>}
-            action={[
-              <Button key="undo" color="inherit" size="small" onClick={this.handleClose}>
-                UNDO
-              </Button>,
-            ]}
-            className={classes.snackBar}
-            transitionProps={{ style: { transform: 'translateY(0px) !important' } }}
+            message={<span id="snackbar-fab-message-id">Archived</span>}
+            action={
+              <Button color="secondary" size="small" onClick={this.handleClose}>
```

Comment:
@oliviertassinari "inherit" was chosen to match the example in the spec. It has better contrast, and is visually distinct from the fab.

Reply:
@mbrookes Ohh. I was wondering. I do agree that the contrast is poor right now.

Reply:
No worries: https://github.com/mui-org/material-ui/commit/f58630bfb71d0db11c3621f86d6e13cdabe8f271